sklearn/metrics/classification.py

```diff
@@ -1563,8 +1564,10 @@ def log_loss(y_true, y_pred, eps=1e-15, normalize=True, sample_weight=None):
         Ground truth (correct) labels for n_samples samples.
 
     y_pred : array-like of float, shape = (n_samples, n_classes)
+             or (n_samples, )
         Predicted probabilities, as returned by a classifier's
-        predict_proba method.
+        predict_proba method. If ``y_pred.shape = (n_samples, )``
+        the score inferred is that of the positive class.
```

Comment:
I know I suggested to add this but wdyt about rewording this like

"the probabilities provided are assumed to be that of the positive class"